### PR TITLE
feat(feat/i18n-hw-card): new features in page HomeworkCard.vue

### PR DESCRIPTION
**Pull request by Misawai**
text in HomeworkCard.vue has been translated.

### CODE REVIEW

File: src/i18n/chinese.ts [hunk 1]
```diff
@@ -120,4 +120,35 @@ export default {
...
+          due: "至",
+        },
+        problems: {
+          text: "題目",
+          id: "序號",
+          pid: "題號",
+          name: "題名",
+          quota: "額度",
+          score: "得分",
+          stats: "狀態",
```
Comment (uier) [inline src/i18n/chinese.ts:143]: The "stats" here means 統計數據, I suggest to translate it into 統計 or 統計數據

File: src/i18n/chinese.ts [hunk 2]
```diff
@@ -120,4 +120,35 @@ export default {
...
+          copycat: "抄襲",
+        },
+        description: {
+          text: "說明",
+          edit: "編輯",
+          stats: "狀態",
```
Comment (uier) [inline src/i18n/chinese.ts:149]: same here

File: src/i18n/english.ts [hunk 3]
```diff
@@ -119,4 +119,35 @@ export default {
       back: "Back",
     },
   },
+  components: {
+    hw: {
+      hwCard: {
+        statusLabel: {
+          running: "RUNNING",
+          nStart: "NOT START",
```
Comment (uier) [inline src/i18n/english.ts:127]: same here

**Review by uier — CHANGES_REQUESTED** [review]

File: src/i18n/chinese.ts (hunk 1 above)
Comment (Misawai) [reply src/i18n/chinese.ts:143]: changed.

File: src/i18n/chinese.ts [hunk 4]
```diff
@@ -120,4 +120,35 @@ export default {
       back: "返回",
     },
   },
+  components: {
+    hw: {
+      hwCard: {
+        statusLabel: {
+          running: "進行中",
+          nStart: "未開始",
```
[the remark this answers is not in the document]
Comment (Misawai) [reply src/i18n/chinese.ts:128]: changed.

File: src/i18n/chinese.ts (hunk 2 above)
Comment (Misawai) [reply src/i18n/chinese.ts:149]: changed

File: src/i18n/english.ts (hunk 3 above)
Comment (Misawai) [reply src/i18n/english.ts:127]: changed

**Pull request merged**